src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -275,6 +275,31 @@ ucs_status_t uct_rc_verbs_ep_am_short(uct_ep_h tl_ep, uint8_t id, uint64_t hdr,
     return UCS_OK;
 }
 
+ucs_status_t uct_rc_verbs_ep_am_short_iov(uct_ep_h tl_ep, uint8_t id,
+                                          const uct_iov_t *iov, size_t iovcnt)
+{
+    uct_rc_verbs_iface_t *iface = ucs_derived_of(tl_ep->iface, uct_rc_verbs_iface_t);
+    uct_rc_verbs_ep_t *ep       = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
+    uint32_t length             = uct_iov_total_length(iov, iovcnt);
```

Comment:
pls, calculate the IOV total length where it is used, i.e. as a parameter to `UCT_RC_CHECK_AM_SHORT()`
to avoid the overhead for release build